src/components/forms/checkbox/checkbox.module.css

```diff
@@ -4,7 +4,13 @@
   align-items: center;
   gap: 0.5rem;
   align-self: stretch;
-  width: -webkit-fill-available;
+  width: 100%;
+  color: var(--primary-black);
+
+  /* to keep checkbox 20x20 we'll make sure portable text takes 2/3 width */
+  & > p {
+    flex: 2;
```

Comment:
not the best approach maybe, but chose to do it this way to avoid filling DOM with bloat, and to keep PortableText from style props.